shared/js/background/classes/tab-state.js

```diff
@@ -32,6 +32,8 @@ export class TabState {
         this.trackers = {}
         /** @type {null | import('../events/referrer-trimming').Referrer} */
         this.referrer = null
+        /** @type {Object} */
```

Comment:
Could you specify this type in more detail? So we avoid typos with the property names?

Reply:
I expanded it a bit but the keys are dynamically generated